### ISSUE TITLE
Does the inconsistent order of normalized channels and image channels affect the results of the self-supervised models?

### ISSUE THREAD

**Issue by liming-ai** [open-mmlab/mmselfsup#212]
Hi @fangyixiao18 @Jiahao000 @XiaohangZhan @xieenze @YuanLiuuuuuu 

As reported in [this issue](https://github.com/open-mmlab/mmselfsup/issues/205), BGR used in pretraining but RGB used in downstream tasks.

However, I found another mistake that in the configs, the order of the values for normalization are also wrong. e.g., in [moco v2](https://github.com/open-mmlab/mmselfsup/blob/af331b043fa6f81037181a83bf8dd38e28bd65c6/configs/selfsup/_base_/datasets/imagenet_mocov2.py#L4), the order of mean and std are follow `RGB`, while the image read order is `BGR` by default.

Does the inconsistent order of normalized channels and image channels affect the results of the self-supervised models? If so, do we need to re-train all the self-supervised models with correct normalization?

**Comment by YuanLiuuuuuu** [comment]
Yes, this bug will affect the performance of the model, but the impact is marginal. Currently, we are re-training all these models in MMSelfSup and will release all updated checkpoints as soon as possible.

**Comment by liming-ai** [comment]
> Yes, this bug will affect the performance of the model, but the impact is marginal. Currently, we are re-training all these models in MMSelfSup and will release all updated checkpoints as soon as possible.

Glad to know that. Will all future versions of mmselfsup follow the channel read order of RGB? (even using mmcv transforms instead of torchvision) I noticed that the default setting in mmcv is BGR

**Comment by fangyixiao18** [comment]
> Glad to know that. Will all future versions of mmselfsup follow the channel read order of RGB? (even using mmcv transforms instead of torchvision) I noticed that the default setting in mmcv is BGR

Yes，we will follow the channel read order of RGB. As you mentioned, the config files of MMDet or MMSeg are using RGB to train the models, we will keep the consistency with other projects.
